redbox-core/redbox/models/settings.py

```diff
@@ -37,15 +37,26 @@ class ElasticCloudSettings(BaseModel):
 class Settings(BaseSettings):
     """Settings for the redbox application."""
 
-    anthropic_api_key: str | None = None
-    openai_api_key: str = "NotAKey"
-    azure_openai_api_key: str = "NotAKey"
-    azure_openai_endpoint: str | None = None
-
-    openai_api_version: str = "2023-12-01-preview"
+    # azure/gpt-35-turbo-16k
```

Comment:
we could move `openai_api_version_*`, `azure_openai_api_key_*` & `azure_openai_endpoint_*` to the AISettings in future